src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx

```diff
@@ -51,7 +51,7 @@ const ProposedSolutionsList: React.FC<ProposedSolutionsListProps> = ({
 
   return (
     <>
-      {!crReviewed ? (
+      {crReviewed === undefined ? (
```

Comment:
What's going on here

Reply:
if crreviewed is undefined/null, that means the change request hasn't been reviewed at all. if it's true, it's approved, if it's false, it's denied. we can only see the add prposed solution button when a CR hasn't been reviewed yet